config/locales/en/admin.yml

```diff
@@ -5,8 +5,8 @@ en:
     actions:
       actions: Actions
       block_author: "Block author"
-      confirm_action: "Are you sure? %{action} \"%{name}\""
-      confirm_delete: "Are you sure? This action will delete \"%{name}\" and can't be undone."
+      confirm_action: 'Are you sure? %{action} "%{name}"'
+      confirm_delete: 'Are you sure? This action will delete "%{name}" and can''t be undone.'
```

Comment:
I noticed there are some changes in this PR related to escaping and single quotes in translations.

To keep things focused, would it make sense to limit this PR to just the translations needed for the new functionality? 🙏